### DISCUSSION TITLE
Error(Replit): Unable to establish a connection to query-engine-node-api library. It seems there is a problem with your OpenSSL installation!

### DISCUSSION THREAD

**Discussion by AyanavaKarmakar** [prisma/prisma#17621] [Q&A]
I keep getting this error whenever I try to run `npx prisma generate` in Replit.

![image](https://user-images.githubusercontent.com/89210438/215489629-31387054-aa43-4821-9187-50db734dec92.png)

I was able to fix the error before by updating the `replit.nix` file with `pkgs.openssl` like this:

```nix
{ pkgs }: {
    deps = [
        pkgs.yarn
        pkgs.esbuild
        pkgs.nodejs-16_x

        pkgs.nodePackages.typescript
        pkgs.nodePackages.typescript-language-server
      
        pkgs.openssl
    ];
}
```

The error showed up again after [updating my schema](https://github.com/TypeSafe-Travellers/trippins). Not sure what is the problem here. Maybe, I'm missing something.

**Comment by nurul3101** [comment]
Hello @AyanavaKarmakar 👋 

There's a related GitHub Issue:
- https://github.com/prisma/prisma/issues/14073

Can you have a look at the solutions mentioned in the issue and see if that helps?

**Reply by AyanavaKarmakar** [reply]
Hi, @nurul3101! Thanks for reaching out. I've dug through the mentioned issue but unfortunately couldn't fix the error.

**Comment by janpio** [comment]
Uh wow, Prisma on Replit. Can you please open a new bug issue with your information?
I know some people internally will be super happy about this - as Replit also uses Nix as they like to do as well 😆

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
